configs/kd/vkd.yaml

```diff
@@ -1,16 +1,24 @@
-experiment: cifar10_vkd
+experiment: vkd-{kd_model.args.distil_weight}-{kd_model.args.temp}-{student_optimizer.type}-{student_optimizer.args.lr}
 batch_size: 32
-epochs: 100
+epochs: 50
```

Comment:
So we are changing to 50 epochs? It doesn't matter what we choose but I think we should remember this decision.

Reply:
it is changed back. I changed it to 50 because of time, but I think it is better to keep at 100.

50 epochs will probably finish by tomorrow afternoon, while 100 epochs won't finish by tomorrow.